src/ape_compile/__init__.py

```diff
@@ -32,6 +37,10 @@ def warn_deprecate(cls, value):
 
         return None
 
+    @validator("exclude", pre=True)
+    def validate_exclude(cls, value):
+        return value or []
```

Comment:
Is this necessary?

Reply:
define necessary? it makes an empty list in the config not be `None`.. helped me when commenting stuff out without having to delete the whole config.

Reply:
i was doing this:

```
compile:
  exclude:
    # - *thing*
```

and i had the error. I think it should know i just mean empty list here and not `None`, or at least i dont see the harm in allowing this state.

Reply:
ahhhh

you were actively inserting a `None`